autoscale-source-dockerswarm/pom.xml

```diff
@@ -50,17 +50,6 @@
             <groupId>org.slf4j</groupId>
             <artifactId>slf4j-api</artifactId>
         </dependency>
-        <dependency>
-            <groupId>org.slf4j</groupId>
-            <artifactId>slf4j-simple</artifactId>
```

Comment:
Unlike the `autoscale-scaler-dockerswarm` case I don't see any logging in [DeveloperEndpointIT.java](https://github.com/Autoscaler/autoscaler/blob/v1.4.0-source/autoscale-source-dockerswarm/src/test/java/com/hpe/caf/autoscale/source/docker/swarm/DeveloperEndpointIT.java) so I agree this is correct.